src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts

```diff
@@ -79,21 +78,20 @@ export class GeneEvidenceProteomicsComponent {
     }
 
     this.initSRM();
-
     this.initLFQ();
     this.initTMT();
   }
 
-  processDifferentialExpressionData(item: any, data: any, dataYAxisMin: number | undefined, dataYAxisMax: number | undefined, proteomicData: any) {
+  processDifferentialExpressionData(item: any, data: any, range: ChartRange, proteomicData: any) {
     const yAxisMin = item.log2_fc < data.min ? item.log2_fc : data.min;
     const yAxisMax = item.log2_fc > data.max ? item.log2_fc : data.max;
 
-    if (dataYAxisMin === undefined || yAxisMin < dataYAxisMin) {
```

Comment:
No need to check for undefined now that we are using the ChartRange class and it will be initialized.  Now the function just needs to check the log2_fc values.